mix.exs

```diff
@@ -112,8 +113,52 @@ defmodule Phoenix.Mixfile do
 
       "deployment/deployment.md": [group: "Deployment"],
       "deployment/heroku.md": [group: "Deployment"]
+      ]
+      |> Enum.map(fn {file, opts} -> {:"guides/docs/#{file}", opts} end)
+  end
+
+  defp groups_for_modules do
+    [
```

Comment:
Let's document the ones that are not in any group at the top.

Reply:
Speaking about this group, I would `@moduledoc false` the Phoenix.Digester. Most interact with it through the task.